stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-14.27 # Last 8.6.5
+resolver: nightly-2020-12-09
```

Comment:
I wonder why did you update this?

Reply:
Merge error. I very carefully checked to make sure that that wasn't included before I pushed. Apparently I checked wrong.